coderd/database/dbauthz/context.go

```diff
@@ -0,0 +1,35 @@
+package dbauthz
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/coderd/rbac"
+)
+
+type authContextKey struct{}
```

Comment:
Could this just be in `dbauthz.go` instead? Seems small and relative to the package as a whole, not a separate part.